README.md

```diff
@@ -23,7 +23,7 @@ npm install electron-osx-sign --save
 npm install electron-osx-sign -g
 ```
 
-*Note: `electron-osx-sign` will become a dependency of `electron-packager` in a later release for signing apps on OS X. However, please install this package globally for more customization beyond specifying identity and entitlements.*
+*Note: `electron-osx-sign` is a dependency of `electron-packager` as of 6.0.0 for signing apps on OS X. However, please install this package globally for more customization beyond specifying identity and entitlements.*
```

Comment:
It's up to you. MAS support is [already documented](https://github.com/electron-userland/electron-packager/blob/master/NEWS.md) that it will be in 6.0.0.